### DISCUSSION TITLE
Identifying edges

### DISCUSSION THREAD

**Discussion by ramcdona** [elalish/manifold#568] [General]
Is there any way to assign an edgeID to an input mesh that will identify that edge (and its split components) in an output mesh?  I.e. some sort of property tracking for edges instead of vertices and faces?

In my particular case, I'd like to be able to track wing trailing edges.  Since this is a classification problem -- every edge either is or is not a TE, I might be able to do this with vertex properties -- every vertex on a TE gets a property of 1.0 -- every other vertex 0.0.

Interpolation along the TE will keep the property as 1.0 -- any off-TE vertex will get a non-unity property.  I should later be able to traverse the mesh and find all edges where both the start and end vertex have property identically 1.0.  Where a wing meets a body, the dual-value property business should be able to sort things out.

Of course, this would be a lot easier if I could just tag Edges with a property and then query that property later.

**Comment by elalish** [comment]
We don't have any edge properties currently. I'm open to ideas for an API if this seems generically useful.

**Reply by ramcdona** [reply]
I don't know whether this would be generally useful or not.

There doesn't really appear to be a user facing edge API at all.

I suspect this would be a lot of work to add and is probably not worth it unless something else pushes for it.
